lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app/Lighty-RNC.postman_collection.json

```diff
@@ -0,0 +1,354 @@
+						"url": {
+							"raw": "http://127.0.0.1:8888/restconf/data/network-topology:network-topology/topology=topology-netconf/node=node-1",
+							"protocol": "http",
+							"host": [
+								"127",
+								"0",
+								"0",
+								"1"
+							],
+							"port": "8888",
+							"path": [
+								"restconf",
+								"data",
+								"network-topology:network-topology",
+								"topology=topology-netconf",
+								"node=node-1"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Disconnect NETCONF device",
+					"request": {
+						"method": "DELETE",
+						"header": [],
+						"url": {
+							"raw": "http://127.0.0.1:8888/restconf/data/network-topology:network-topology/topology=topology-netconf/node=node-1",
+							"protocol": "http",
+							"host": [
+								"127",
+								"0",
+								"0",
+								"1"
+							],
+							"port": "8888",
+							"path": [
+								"restconf",
+								"data",
+								"network-topology:network-topology",
+								"topology=topology-netconf",
+								"node=node-1"
+							]
+						}
+					},
+					"response": []
+				}
+			]
+		},
+		{
+			"name": "Device data manipulation",
+			"item": [
+				{
+					"name": "GET NETCONF device data",
+					"request": {
+						"method": "GET",
+						"header": [],
+						"url": {
+							"raw": "http://127.0.0.1:8888/restconf/data/network-topology:network-topology/topology=topology-netconf/node=node-1/yang-ext:mount/",
+							"protocol": "http",
+							"host": [
+								"127",
+								"0",
+								"0",
+								"1"
+							],
+							"port": "8888",
+							"path": [
+								"restconf",
+								"data",
+								"network-topology:network-topology",
+								"topology=topology-netconf",
+								"node=node-1",
+								"yang-ext:mount",
+								""
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "PUT  config data on NETCONF device",
+					"request": {
+						"method": "PUT",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n    \"toaster:toaster\": {\n        \"darknessFactor\": 500\n    }\n}",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "http://127.0.0.1:8888/restconf/data/network-topology:network-topology/topology=topology-netconf/node=node-1/yang-ext:mount/toaster:toaster",
+							"protocol": "http",
+							"host": [
+								"127",
+								"0",
+								"0",
+								"1"
+							],
+							"port": "8888",
+							"path": [
+								"restconf",
+								"data",
+								"network-topology:network-topology",
+								"topology=topology-netconf",
+								"node=node-1",
+								"yang-ext:mount",
+								"toaster:toaster"
+							]
+						},
+						"description": "Example request for setting configuration data on NETCONF device. This request is valid if the device does implement toaster yang."
+					},
+					"response": []
+				},
+				{
+					"name": "GET restconf operations",
```

Comment:
This is not a "Device data manipulation"

Reply:
Done, removed the request, it is very similar to others.